pkgs/development/python-modules/dict2css/default.nix

```diff
@@ -0,0 +1,32 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  whey,
+  cssutils,
+  lib,
+}:
+buildPythonPackage rec {
+  pname = "dict2css";
+  version = "0.3.0.post1";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit version;
+    pname = "dict2css";
+    hash = "sha256-icVEwhxMp0csP/+50309km9gYymv23Udwd5npBG3Bxk=";
+  };
+
+  build-system = [ whey ];
+
+  dependencies = [ cssutils ];
+
+  pythonImportsCheck = [ "dict2css" ];
+
+  pythonNamespaces = [ "dict2css" ];
```

Comment:
We otherwise rarely set pythonNamespaces. Do you really think all those packages need that?

Reply:
No, I just copied what I saw in other python packages. I'll check if they can be removed.